frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -186,7 +182,9 @@ function ReportsTable() {
                     <p>
                         Default service:{" "}
                         <strong>
-                            {services?.[0].name.toUpperCase() || ""}
+                            {(services?.length &&
+                                services[0].name.toUpperCase()) ||
```

Comment:
This failing to return should be a fatal error for the page and hit an ErrorBoundary. Any receiver who happens upon this page and gets a 4xx or 5xx will now just see an empty page. Seems like an opportunity to check a single box of our long list for that ErrorBoundary tech debt we have coming up by figuring this one out.

Reply:
Turns out the endpoint doesn't give a 404 for that call being undefined -- this'll be trickier than what this PR is for